### ISSUE TITLE
Support for Ubuntu Jammy

### ISSUE THREAD

**Issue by romanyacik** [vegastrike/Vega-Strike-Engine-Source#680]
Ubuntu 22.04 is not supported. build.sh exits with the following error:
SEND_WARNING!! Unsupported Debian derivative.

**Comment by LifWirser** [comment]
Could aa attempt to build the "hard way" ie: mkdir build && cd build]ret\
cmake ../ [re]
make [ret]
|

**Comment by romanyacik** [comment]
> Could an attempt to build the "hard way" be made ie: mkdir build && cd build [ret] cmake . ./engine[ret] make [ret] |

Warning message persists, but it was still built. script/package fails with the message "Please build Vega Strike first."

**Comment by stephengtuggy** [comment]
Okay. So it builds, just doesn't package up the installer properly. That's good to know.

Can you run it?

**Comment by romanyacik** [comment]
"vegasettings" and "vegastrike-engine where built into the bin directory. vegasettings fails with the following message: 
"Error: Failed to find Version.txt anywhere." I have forgotten how to specify the data directory for vegastrike-engine

**Comment by stephengtuggy** [comment]
vegastrike-engine -d followed immediately by the path to the data directory (no space after `-d`). IIRC, one way to achieve this, at least in bash or similar, is to cd into the data directory, and then run `vegastrike-engine -d$(cwd)`

**Comment by evertvorster** [comment]
Are we not installing scripts that automatically set this directory?

**Comment by stephengtuggy** [comment]
@evertvorster Not necessarily. The installer doesn't work yet on Ubuntu 22.04 "Jammy", so we're having to improvise a little at the moment.

**Comment by romanyacik** [comment]
> vegastrike-engine -d followed immediately by the path to the data directory (no space after `-d`). IIRC, one way to achieve this, at least in bash or similar, is to cd into the data directory, and then run `vegastrike-engine -d$(cwd)`

Thank you. It runs.

**Comment by stephengtuggy** [comment]
Thanks, @romanyacik , for testing that out for us.

**Comment by BenjamenMeyer** [comment]
We'll need this fixed for 0.9.x, and we should fix it for 0.8.x too.

**Comment by stephengtuggy** [comment]
Relates to #645

**Comment by stephengtuggy** [comment]
Oops, this issue should not have been closed yet. We do not yet have support for Jammy on 0.8.x.

**Comment by stephengtuggy** [comment]
> Oops, this issue should not have been closed yet. We do not yet have support for Jammy on 0.8.x.

Now we do!

**Comment by stephengtuggy** [comment]
@BenjamenMeyer can we close this issue? We have had Jammy support for some time now, AFAIK.

**Comment by royfalk** [comment]
I'm pretty sure that due to boost now requiring 1.75, we no longer have it again :)
